docs-source/content/samples/simple/elastic-stack/weblogic-domain/_index.md

```diff
@@ -0,0 +1,335 @@
+---
+title: "WebLogic Domain"
+date: 2019-02-23T17:32:31-05:00
+weight: 6
+description: "Sample for configuring fluentd to export domain logs to ELK."
+---
+
+
+#### Overview
+This page describes to how to configure a WebLogic domain to use fluentd to send log information to ELK.
+
+The general mechanism of how this works is:
+* fluentd runs as a separate container in the admin and managed server pods
```

Comment:
these don't seem to be formatting right, might need a blank line before them - or maybe it is just me

Reply:
Either use "FluentD" here or "`fluentd`" in backticks

Reply:
Actually, the product name is Fluentd (this is how I see it in their documentation). It should either be always capitalized or in back ticks, as Mark specified.

Reply:
done in next push